src/utils/SVGIcons.ts

```diff
@@ -0,0 +1,24 @@
+import { $$, Dom } from './Dom';
+
+declare var require: (svgPath: string) => string;
+
+export class SVGIcons {
+  public static search = require('svg/search');
+  public static more = require('svg/more');
+  public static facetLoading = require('svg/facet-loading');
+  public static checkboxMoreValues = require('svg/checkbox-hook-exclusion-more');
+  public static arrowUp = require('svg/arrow-up');
+  public static arrowDown = require('svg/arrow-down');
+
+  private static parser = new DOMParser();
+
+  public static addClassToSVGInContainer(svgContainer: HTMLElement, classToAdd: string) {
```

Comment:
I had to do this because I can't use the `addClass` method we have in the `Dom` class since the `className` attribute on an `SVG` is not a string, it's a `SVGAnimatedString`. Therefore when trying to call `this.el.className.match` it fails. I will add a class to perform this operation on an `SVG` in another class.